test/algconvergence/linear_nonlinear_krylov_tests.jl

```diff
@@ -7,7 +7,7 @@ let N = 20
     A = diagm(-1 => du, 0 => dd, 1 => du)
     _f = (u, p, t) -> A * u - u .^ 3
     _f_ip = (du, u, p, t) -> (mul!(du, A, u); du .-= u .^ 3)
-    _jac = (A, u, p, t) -> A - 3 * diagm(0 => u .^ 2)
+    _jac = (J, u, p, t) -> A - 3 * diagm(0 => u .^ 2)
```

Comment:
It's the old op matrix, which probably shouldn't be provided for OOP update_coefficients since unlike the in place version it doesn't need to be overwritten: https://github.com/SciML/SciMLOperators.jl/issues/202

This PR just gets things to work with the current signature of update_coefficienfs